compiler/rustc_trait_selection/src/traits/select/mod.rs

```diff
@@ -2,6 +2,12 @@
 //!
 //! [rustc dev guide]: https://rustc-dev-guide.rust-lang.org/traits/resolution.html#selection
 
+// FIXME: This should be temporal. The `map` field in ProvisionalEvaluationCache should be changed to
```

Comment:
Temporal -> temporary

Or better, just remove that sentence

Reply:
Thanks!